CHANGES_NEXT_RELEASE

```diff
@@ -3,3 +3,4 @@
 - Add: support for HTTPS requests toward CB (if IOTA_CB_URL is used) (#578)
 - Add: support HTTPS requests toward IOTAManager (if IOTA_IOTAM_URL is used) (#578)
 -  Fix: documentation and examples, using correctly "attributes" field in the Configuration API instead of "active" (#558)
+- Fix: incomplete HTTPS support for subscriptions (#594)
```

Comment:
Self-reference to the PR? :) 

Maybe you mean 594 -> 593 ?

Reply:
sorry :)

Reply:
fixed by a7cb7a0